### DISCUSSION TITLE
Send to Copilot Context Menu action

### DISCUSSION THREAD

**Discussion by MTGMAD** [logancyang/obsidian-copilot#746] [Ideas]
It would be highly beneficial to expand the context menu functionality beyond the “Copilot: Summarize Selection” option. An enhancement that after you select text, then you could just right-click, and choose “Send to Copilot” would streamline the workflow significantly. Currently, I often resort to manually copying and pasting text into the Copilot chat mode to further manipulate it within the Copilot sidebar. Implementing a direct “Send to Copilot” action would save time and improve the overall user experience and efficiency. 

Thank you for taking into consideration

**Comment by logancyang** [comment]
By "send to Copilot", would you like to pass the selection to your chat input box without actually sending to the LLM? Since in most cases I guess there needs to be some more instructions to add to the prompt.

**Reply by MTGMAD** [reply]
Correct, just pass it to the chat input box and not to the LLM, so prompt editing can happen before it is sent to the LLM.  Thank you for clarifying.
